packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -23,13 +23,18 @@ interface EntityRecordResolution< RecordType > {
 	status: Status;
 }
 
+interface Options {
+	__experimentalEnabled: boolean;
+}
+
 /**
  * Resolves the specified entity record.
  *
  * @param  kind     Kind of the requested entity.
  * @param  name     Name of the requested  entity.
  * @param  recordId Record ID of the requested entity.
  *
```

Comment:
Super nitpick: we don't need this line:
```suggestion
```

I see it's experimental for now. Is it the reason you didn't document it here? Is it something that gets normally reflected in the documentation in other places?

Reply:
Good call, I added a relevant docstring